client/CMT.cpp

```diff
@@ -297,6 +314,11 @@ int main(int argc, char * argv[])
 	logConfig->configure();
 	logGlobal->debug("settings = %s", settings.toJsonNode().toJson());
 
+	// New attrs

```

Comment:
It will be new forever? No such comments, please.

Reply:
If you need some comment for a part of code, you can move this code to a function as an option and use function name to express what comment does. This makes it readable and reduces size of parent function making it more readable as well.

Reply:
Not sure if we should introduce a function here so at least consider making your comment more meaningful. Or removing it.